src/components/visualizations/chart/ChartTransformation.tsx

```diff
@@ -115,12 +119,13 @@ export default class ChartTransformation extends React.Component<
             executionRequest: { afm, resultSpec },
             executionResponse,
             executionResult: { data, headerItems },
-            config,
             onDataTooLarge,
             onNegativeValues,
             pushData,
         } = props;
 
+        const chartConfig = this.getChartConfig(props);
```

Comment:
This change should be unit tested. It seems to be covered by Storybooks but there should be unit tests as well.

Reply:
added new tests